projects/components/src/navigation/navigation-list.component.ts

```diff
@@ -35,64 +35,68 @@ import { IconSize } from '../icon/icon-size';
           <htc-link class="link" [url]="footerItem.url">
-            <htc-icon *ngIf="isCollapsed" [icon]="footerItem.icon" size="${IconSize.Small}"></htc-icon>
-            <htc-label *ngIf="!isCollapsed" [label]="footerItem.label"></htc-label>
+            <htc-icon *ngIf="this.collapsed" [icon]="footerItem.icon" size="${IconSize.Small}"></htc-icon>
+            <htc-label *ngIf="!this.collapsed" [label]="footerItem.label"></htc-label>
           </htc-link>
         </div>
       </div>
     </nav>
   `
 })
 export class NavigationListComponent {
-  private static readonly COLLAPSED_PREFERENCE: string = 'navigation-list.collapsed';
   @Input()
   public navItems: NavItemConfig[] = [];
 
   @Input()
   public footerItems?: FooterItemConfig[];
 
+  @Input()
+  public collapsed?: boolean = false;
+
+  @Input()
+  public resizable?: boolean = true;
+
+  @Output()
+  public readonly collapsedChange: EventEmitter<boolean> = new EventEmitter();
+
   public readonly activeItem$: Observable<NavItemLinkConfig | undefined>;
-  public readonly isCollapsed$: Observable<boolean>;
-  public readonly resizeIcon$: Observable<IconType>;
 
-  public constructor(
-    private readonly navigationService: NavigationService,
-    private readonly preferenceService: PreferenceService
-  ) {
+  public constructor(private readonly navigationService: NavigationService) {
     this.activeItem$ = this.navigationService.navigation$.pipe(
       startWith(this.navigationService.getCurrentActivatedRoute()),
       map(() => this.findActiveItem(this.navItems))
     );
-    this.isCollapsed$ = this.preferenceService.get(NavigationListComponent.COLLAPSED_PREFERENCE, false);
-    this.resizeIcon$ = this.isCollapsed$.pipe(
-      map(isCollapsed => (isCollapsed ? IconType.TriangleRight : IconType.TriangleLeft))
-    );
   }
 
   public navigate(item: NavItemLinkConfig): void {
-    this.navigationService.navigateWithinApp([item.matchPaths[0]]);
+    this.navigationService.navigateWithinApp(item.matchPaths);
+  }
+
+  public toggleView(): void {
+    if (this.resizable) {
+      this.collapsed = !this.collapsed;
+      this.collapsedChange.emit(this.collapsed);
+    }
   }
 
-  public setCollapsed(isCollapsed: boolean): void {
-    this.preferenceService.set(NavigationListComponent.COLLAPSED_PREFERENCE, isCollapsed);
+  public getResizeIcon(): IconType {
+    return this.collapsed ? IconType.TriangleRight : IconType.TriangleLeft;
   }
 
   private findActiveItem(navItems: NavItemConfig[]): NavItemLinkConfig | undefined {
     return navItems
       .filter((item): item is NavItemLinkConfig => item.type === NavItemType.Link)
       .find(linkItem =>
-        linkItem.matchPaths.some(matchPath =>
-          this.navigationService.isRelativePathActive([matchPath], this.navigationService.rootRoute())
-        )
+        this.navigationService.isRelativePathActive(linkItem.matchPaths, this.navigationService.rootRoute())
```

Comment:
This seems like a bug - match paths should be `OR'd` (hence the previous `some`)

Reply:
For settings, the navigation match path would be `['settings', 'users']`. `isRelativePathActive` check fails for such a case. Why do we want to `or` this?

Reply:
that should be expressable as `['settings/users']`

Reply:
I see. Right not there are none and this use case was also not clear from the api. Updating it

Reply:
👍 I think currently the IA does not have this use case, but it comes and goes so retaining support makes sense.